data/Templates/Hl7v2/DataType/_HDOrganization.liquid

```diff
@@ -5,6 +5,9 @@
     },
     {
         "value":"{{ HD.2.Value }}",
+        "type":
```

Comment:
Because the mapping table doesn't point out the exact mapping relation(like what datatype should this attribute have), and we want to keep as many information as we can, so we just put a type attribute here without any children properties.
What's your opinion Ranvijay? Should we keep or remove it?